less/_mixins.less

```diff
@@ -48,18 +39,23 @@
 
 // hidden (accessible by screen readers)
 .hidden {
```

Comment:
I updated the contents of that one to reflect what is in the H5BP. Safe to assume their snippet is pretty well tested